host/xtest/pkcs11_1000.c

```diff
@@ -2565,3 +2565,218 @@ static void xtest_pkcs11_test_1011(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(pkcs11, 1011, xtest_pkcs11_test_1011,
 		 "PKCS11: Test Find Objects");
+
+static void xtest_pkcs11_test_1012(ADBG_Case_t *c)
+{
+	CK_RV rv = CKR_GENERAL_ERROR;
+	CK_SLOT_ID slot = 0;
+	CK_SESSION_HANDLE session = CK_INVALID_HANDLE;
+	CK_FLAGS session_flags = CKF_SERIAL_SESSION | CKF_RW_SESSION;
+	CK_OBJECT_HANDLE obj_hdl = CK_INVALID_HANDLE;
+	size_t i = 0;
+
+	CK_OBJECT_CLASS objClass = CKO_DATA;
```

Comment:
prefer snake case, here and there, i.e. s/objClass/obj_class/g. for consistency in xtest implementation.

Reply:
OK

Reply:
Did bunch of similar fixes.